adidas-ascii-logo/javascript/README.md

```diff
@@ -0,0 +1,54 @@
+# adidas ASCII logo: JavaScript
+
+Implementation to resolve the [adidas ASCII logo challenge][adidas-ascii-logo] with JavaScript ES6+.
+
+## Requirements and dependencies
+
+[NodeJS/NPM][nodejs] are required to work with the repository.
+
+## Installation and running
+
+Install the dependendies:
+
+```
+npm install
+```
+
+### Testing
+
+To be sure that the altorithm is correct, there are several tests which checks the requirements.
```

Comment:
Maybe we can rephrase to `there are several tests to validate the requirements are met`